### ISSUE TITLE
cudashift

### ISSUE THREAD

**Issue by TryHard-LL** [deepcs233/TIN#5]
Hi，”cudashift" must be installed？

**Comment by deepcs233** [comment]
Yes, or you may need to implement the function "shift the data by the tmeporal dimension" in naive Pytorch by yourself to replace the cuda kernel.

**Comment by TryHard-LL** [comment]
when I use "bash make.sh", I can't compile it successfully...
such as "RuntimeError: Error compiling objects for extension"

**Comment by deepcs233** [comment]
maybe you can check the version of PyTorch and CUDA. For me CUDA 10.1 and Pytorch 1.4.0 works well
